js/gdm/authPrompt.js

```diff
@@ -669,18 +669,23 @@ const AuthPrompt = new Lang.Class({
         return unlockCode;
     },
 
-    _showPasswordResetPrompt: function() {
+    _getCustomerSupportEmail: function() {
         let keyFile = this._ensureCustomerSupportFile();
-        let customerSupportEmail = null;
 
         try {
-            customerSupportEmail = keyFile.get_locale_string(CUSTOMER_SUPPORT_GROUP_NAME,
+            return keyFile.get_locale_string(CUSTOMER_SUPPORT_GROUP_NAME,
                                                              CUSTOMER_SUPPORT_KEY_EMAIL,
```

Comment:
Indentation seems off for the parameters

Reply:
This indentation is still off

Reply:
Ah! It's fixed in the following commit. Nevermind then, I will probably "fix" while doing the next rebase. Let's not go for another review cycle because of this nit :-)